### ISSUE TITLE
Add more arguments for Cosine_with_restarts scheduler

### ISSUE THREAD

**Issue by Azurlifrfa** [Linaqruf/kohya-trainer#25]
Please add some parameters in your finetuner, like Minimum and maximum learning rate, rate of increasing/decreasing learning rate per cycle, numbers of epochs per cycle and warmup steps per cycle, like in Aria1th's monkey patch          
>https://github.com/aria1th/Hypernetwork-MonkeyPatch-Extension

**Comment by Linaqruf** [comment]
I think you need to open this issue in main repo instead, because this repo just a fork for colab notebook implementation.
https://github.com/kohya-ss/sd-scripts/

but sure, i'll try my best.

**Comment by Linaqruf** [comment]
Hi, now this repo support lr scheduler power for polynomial and num cycle for cosine with restarts, thanks to kohya updating her repo.
